uint8 SERIAL = 0
uint8 BLE = 1

uint8 control_interface